nservicebus/sqlserver/service-control.md

```diff
@@ -0,0 +1,37 @@
+---
+title: ServiceControl and Multi-Instance Mode
+summary: ServiceControl and SQL Server transport multi-instance mode configuration guidance 
+reviewed: 2016-06-24
+tags:
+ - SQL Server
+ - Transports
+ - ServiceControl
+---
+
+
+## Overview
+Even though it is [recommended that all SQL Server transport queue tables are stored in a single SQL Server catalog](/#deployment-considerations), it is possible to use ServiceControl to monitor a multi-instance deployment of the SQL Server transport. A requirement for such a configuration is that all endpoints share `error` and `audit` queues and that these queues are stored in the same catalog as ServiceControl queues. Other queues used by individual endpoints may be stored in different SQL Server instances. The following diagram shows an example system configuration:
```

Comment:
Maybe "multi-catalog"? I think we should stay consistent in a single statement and talk about instances or catalogs. Actually I think that an ambiguous term 'database' would be a good fit here.